python_modules/dagster/dagster/_core/pipes/subprocess.py

```diff
@@ -51,6 +53,7 @@ def __init__(
         context_injector: Optional[PipesContextInjector] = None,
         message_reader: Optional[PipesMessageReader] = None,
         forward_termination: bool = True,
+        forward_stdio: bool = True,
```

Comment:
I've added this parameter so it will be possible to disable log forwarding in tests. 
This is because otherwise it becomes really hard to determine whether a log line is coming from the Pipes external process or was it logged from Dagster from the new `log_stdio` message.